### PR TITLE
Compress images using sharp

### PR DESCRIPTION
**Pull request by yenaing-oo**
An alternative method to compressing images, that does not rely on webpack.

### CODE REVIEW

**Comment by yenaing-oo** [comment]
Here is a comparison of the original image file sizes vs the compressed ones:

Original
image

Compressed
image

File: scripts/compress_images.js
Comment (danloveg) [inline scripts/compress_images.js:1]: Nice, you wrote this in Javascript!

I do have a comment, can you add a console.log at the start and end so there's more feedback in the logs about when this script starts and ends?

**Review by danloveg — CHANGES_REQUESTED** [review]
I like this, it's simple and works in development and production.

I also like if we accidentally forget to link to the `.webp` file that the original file will still load fine.

File: scripts/compress_images.js
Comment (yenaing-oo) [reply scripts/compress_images.js:1]: Absolutely.

Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/264/commits/6086edc73c6cd7fb157737e36fddb86a3c82459e

**Review by danloveg — DISMISSED** [review]
This all looks great.

This opens the door to migrate this code to a custom webpack plugin. For the time being, though, there's no need to do anything more complex than this.

**Review by danloveg — APPROVED** [review]

**Pull request merged**